README.md

```diff
@@ -125,7 +142,15 @@ Write code to do the following:
 
 Write your code here:
 ```ruby
-# code here
+lamp =
+{
+  wishes_remaining: ["wish1", "wish2", "wish3"]
+  genie: ["genie"]
+}
+lamp.genie.slice[1,2]
+lamp[:wishes_remaining]="jafar"
+
```

Comment:
This question was giving you a chance to share your knowledge of ActiveRecord and how it helps us to manage relationships.  I can see where you were going with this code, but it ignores the provided Genie and Lamp classes, not taking advantage of their ties to the database tables and the methods they provide.